docs/user-guide/configure-zowe-zosmf-workflow.md

```diff
@@ -85,19 +85,21 @@ This workflow is in three parts:
 ### Define variables
 
 This workflow step includes the list of Zowe variables for configuration and as well as sub-steps.
-
+ 
 1. Expand **Define variables** to display the following screen:  
 
 ![Step1: Define variables](../images/zosmf/workflow-defineVariables.png)
 
 2. Under **Input Variables**, click **Define the main variables**.
 3. Click the **Perform** tab. The input variables are displayed by category. 
-4. Customize workflow variables in the fields provided.  
-   **Example:**
+4. Customize workflow variables in the fields provided. Single-service deployment mode is enabled
```

Comment:
Is it worth explaining that multi-service deployment is enabled when this box is left unticked? Perhaps that is obvious...